src/platform/src/Bridge/Mistral/Mistral.php

```diff
@@ -22,6 +22,7 @@ final class Mistral extends Model
     public const CODESTRAL = 'codestral-latest';
     public const CODESTRAL_MAMBA = 'open-codestral-mamba';
     public const MISTRAL_LARGE = 'mistral-large-latest';
+    public const MISTRAL_MEDIUM = 'mistral-medium-latest';
```

Comment:
Can you review all models please? :angel: 

See

```bash
curl -s -q --location "https://api.mistral.ai/v1/models" --header 'Content-Type: application/json' --header 'Accept: application/json' --header "Authorization: Bearer $MISTRAL_API_KEY" | jq | grep '"id"'
```

I.e. `mistral-3b-latest` does not exist, it should be `ministral-3b-latest` instead.

Reply:
added corresponding changes in 9b94730